app/presenters/tree_node/middleware_server.rb

```diff
@@ -0,0 +1,5 @@
+module TreeNode
+  class MiddlewareServer < Node
+    set_attribute(:title) { |object| "#{object.name.presence} - #{object.host_name}" }
```

Comment:
* Use `@object` instead of `object` and so remove `|object|`
* Use `String[]#join` instead of the interpolation